client/dist/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <link rel="stylesheet" type="css" href="../styles.css"/>
     <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
-  <script defer src="booking.js?15eb6958c0b7f85d502f"></script></head>
+  <script defer src="booking.js?a01fd52f277633dcdd0a"></script></head>
```

Comment:
what is this defer sorcery?

Reply:
i didn't write that but i think it's there because webpack renders my html from a template in my assets dir...not sure, need to look into it.